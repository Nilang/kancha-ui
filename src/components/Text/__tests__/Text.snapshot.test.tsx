import React from 'react'
import { render } from 'react-native-testing-library'

import Text from '../Text'

describe('Container', () => {
  it('renders correctly', () => {
    const tree = render(<Text>Hello World</Text>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
