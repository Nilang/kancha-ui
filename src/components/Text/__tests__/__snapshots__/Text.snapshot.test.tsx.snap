// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly with all props 1`] = `
<Text
  style={
    Object {
      "color": "#FFFFFF",
      "fontSize": 32,
      "fontWeight": "bold",
      "opacity": 0.9,
      "paddingBottom": 10,
      "textAlign": "center",
      "textTransform": "uppercase",
    }
  }
>
  Hello World
</Text>
`;

exports[`Container renders correctly with default padding bottom 1`] = `
<Text
  style={
    Object {
      "paddingBottom": 15,
    }
  }
>
  Hello World
</Text>
`;

exports[`Container renders correctly with no props 1`] = `
<Text
  style={Object {}}
>
  Hello World
</Text>
`;
